src/containers/AppNewContainer/AppNewContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import { connect } from "react-redux";
+import { push } from "react-router-redux";
+import { Dispatch } from "redux";
+
+import actions from "../../actions";
+import AppNew from "../../components/AppNew";
+import { IChartVersion, IStoreState } from "../../shared/types";
+
+interface IRouteProps {
+  match: {
+    params: {
+      repo: string;
+      id: string;
+      version?: string;
+    };
+  };
+}
+
+function mapStateToProps({ apps, charts }: IStoreState, { match: { params } }: IRouteProps) {
+  return {
+    chartID: `${params.repo}/${params.id}`,
+    chartVersion: params.version,
+    selected: charts.selected,
+  };
+}
+
+function mapDispatchToProps(dispatch: Dispatch<IStoreState>) {
+  return {
+    deployChart: (version: IChartVersion, releaseName: string, namespace: string, values?: string) =>
+      dispatch(actions.charts.deployChart(version, releaseName, namespace, values)),
+    getChartValues: (id: string, version: string) =>
+    dispatch(actions.charts.getChartValues(id, version)),
```

Comment:
I think it's more readable if this is indented, I thought prettify would have done that though?